translations/strings.yaml

```diff
@@ -443,10 +443,81 @@ mobileOnboarding:
   alreadyASubscriber: Already a subscriber?
 
 inAppAuth:
+  emailInputPlaceholder: Enter email
   changeEmailLink: Change email
   signInButton: Sign in
   signInSubtitle: Continue to Mozilla VPN by signing in with your Firefox account
+  createPasswordLabel: Create password
```

Comment:
What is this? A button, something else?

Reply:
It's a label above the password creation input.

Reply:
OK, could you add a comment just to be on the safe side?